test/audit/AuditingServiceSpec.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 HM Revenue & Customs
+ * Copyright 2019 HM Revenue & Customs
```

Comment:
You have a failing test in this spec. It's related to the change in auditing. The change Dave made in vat-returns was to remove the extra fields from the JSON in the val `expected` in the test. I'm awaiting confirmation from a TxM rep about whether we would need to update our existing TxM pages but we can do it now for our purposes right now to get the tests passing.